### PR TITLE
added benchmark to lruAccounts write function

### PR DESCRIPTION
**Pull request by algonathan**
## Summary
lruAccounts write function benchmark

### CODE REVIEW

File: ledger/lruaccts_test.go [hunk 1]
```diff
@@ -195,3 +196,31 @@ func TestLRUAccountsOmittedPendingWrites(t *testing.T) {
...
+			addr:        basics.Address(digest),
+			round:       basics.Round(i),
+			rowid:       int64(i),
+			accountData: basics.AccountData{MicroAlgos: basics.MicroAlgos{Raw: microAlgos}},
+		}
+	}
+
+	baseAccts := make([]lruAccounts, b.N)
+	for i := 0; i < b.N; i++ {
+		baseAccts[i].init(logging.TestingLog(b), 10, 5)
```
Comment (tsachiherman) [inline ledger/lruaccts_test.go:216]: In order to get realistic timings, you'll need to fill this data structure with dummy data.
Testing the performance again an empty datastructure is less than likly to represent a typical load.

Comment (algonathan) [reply ledger/lruaccts_test.go:205]: I see. 
i've changed the coed to generate 5k accounts for the benchmark, and made certain to fill the `baseAccts` with 95k different `persistedAccountData` before running the benchmark.

File: ledger/lruaccts_test.go [hunk 2]
```diff
@@ -195,3 +196,40 @@ func TestLRUAccountsOmittedPendingWrites(t *testing.T) {
...
+			baseAcct.write(account)
+		}
+	}
+}
+
+func generatePersistedAccountData(startRound, endRound int) []persistedAccountData {
+	accounts := make([]persistedAccountData, endRound-startRound)
+	for i := startRound; i < endRound; i++ {
+
+		digest := crypto.Hash([]byte{byte(i)})
```
Comment (tsachiherman) [inline ledger/lruaccts_test.go:224]: This would create many entries.. but only 256 unique accounts.
I don't believe it was your intent, right ?

Comment (tsachiherman) [reply ledger/lruaccts_test.go:224]: Creating overlapping account between the two groups is fine, of course

Comment (algonathan) [reply ledger/lruaccts_test.go]: You are correct, thank you for noticing!

I've decided to avoid overlapping accounts, I hope this isn't an issue.

**Pull request merged**